docs/concepts/databases/elasticsearch.md

```diff
@@ -269,11 +311,144 @@ Elasticsearch managed by KubeDB can be monitored with builtin-Prometheus and Cor
 - [Monitor Elasticsearch with builtin Prometheus](/docs/guides/elasticsearch/monitoring/using-builtin-prometheus.md)
 - [Monitor Elasticsearch with CoreOS Prometheus operator](/docs/guides/elasticsearch/monitoring/using-coreos-prometheus-operator.md)
 
+### spec.configSource
+
+`spec.configSource` is an optional field that allows users to provide custom configuration for Elasticsearch. This field accepts a [`VolumeSource`](https://github.com/kubernetes/api/blob/release-1.11/core/v1/types.go#L47). So you can use any kubernetes supported volume source such as `configMap`, `secret`, `azureDisk` etc. To learn more about how to use a custom configuration file see [here](/docs/guides/elasticsearch/custom-config/overview.md).
+
+### spec.podTemplate
+
+KubeDB allows providing a template for database pod through `spec.podTemplate`. KubeDB operator will pass the information provided in `spec.podTemplate` to the StatefulSet created for Elasticsearch database.
+
+KubeDB accept following fields to set in `spec.podTemplate:`
+
+- annotations (pod's annotation)
+- controller.annotations (statefulset's annotation)
+- spec:
+  - env
```

Comment:
`args` missing

Reply:
we don't pass args to elasticsearch pod.